src/components/calcite-switch/calcite-switch.scss

```diff
@@ -1,149 +1,146 @@
 }
 
 :host([disabled]) {
-  opacity: var(--calcite-ui-opacity-disabled);
-  pointer-events: none;
-  cursor: default;
+  @apply opacity-disabled pointer-events-none cursor-default;
 }
 // focus styles
 :host {
-  @include focus-style-base();
-  width: auto;
+  @apply focus-base w-auto;
 }
 
 :host(:focus) {
-  @include focus-style-outset();
+  @apply focus-outset;
 }
 
 .track {
-  position: relative;
-  display: inline-block;
-  vertical-align: top;
-  width: var(--calcite-switch-track-width);
-  height: var(--calcite-switch-track-height);
-  background-color: var(--calcite-ui-foreground-2);
-  border-radius: 30px;
-  border: 1px solid var(--calcite-ui-border-2);
-  pointer-events: none;
-  transition: all $transition;
+  @apply relative 
+    inline-block 
+    align-top 
+    bg-foreground-2 
+    border 
+    border-solid 
+    border-color-2 
+    box-border
+    pointer-events-auto
+    transition-all
+    duration-150
+    ease-in-out
+    rounded-full;
 }
 
 .handle {
-  position: absolute;
-  display: block;
-  width: var(--calcite-switch-handle-size);
-  height: var(--calcite-switch-handle-size);
+  @apply absolute 
+    block 
+    right-auto 
+    bg-foreground-1 
+    border-2 
+    border-solid 
+    border-color-input
+    transition-all
+    duration-150
+    ease-in-out
+    rounded-full;
   top: -1px;
   left: -1px;
-  right: auto;
-  background-color: var(--calcite-ui-foreground-1);
-  border-radius: 30px;
-  border: 2px solid var(--calcite-ui-border-input);
-  pointer-events: none;
-  transition: all $transition;
 }
 
 :host(:hover),
 :host(:focus) {
   .handle {
-    border-color: var(--calcite-switch-hover-handle-border);
-    box-shadow: inset 0 0 0 1px var(--calcite-switch-hover-handle-border);
-    right: auto;
+    border-color: theme("backgroundColor.brand");
+    box-shadow: inset 0 0 0 1px theme("backgroundColor.brand");
+    @apply right-auto;
   }
 }
 
 :host([switched]) {
   .track {
-    background-color: var(--calcite-switch-switched-track-background);
-    border-color: var(--calcite-switch-switched-track-border);
+    background-color: theme("backgroundColor.brand");
+    border-color: theme("borderColor.color-brand");
   }
   .handle {
+    @apply left-auto;
     right: -1px;
-    left: auto;
-    border-color: var(--calcite-switch-switched-handle-border);
+    border-color: theme("borderColor.color-brand");
   }
 }
 
 :host([switched]:hover) {
   .track {
-    background-color: var(--calcite-switch-switched-hover-track-background);
-    border-color: var(--calcite-switch-switched-hover-track-border);
+    background-color: theme("backgroundColor.brand");
+    border-color: theme("backgroundColor.brand");
   }
   .handle {
-    border-color: var(--calcite-switch-switched-hover-handle-border);
-    box-shadow: inset 0 0 0 1px var(--calcite-switch-switched-hover-handle-border);
+    border-color: theme("borderColor.color-brand-hover");
+    box-shadow: inset 0 0 0 1px theme("borderColor.color-brand-hover");
```

Comment:
☝️ same here for these hover colors.  Should we add these to the tailwind config so we can use @apply?

Reply:
my bad, this exists, just pushed an update.